rdftk_ontology/src/owl/mod.rs

```diff
@@ -31,22 +31,31 @@ pub enum HeaderProperty {
 
 #[derive(Clone, Debug)]
 pub struct Ontology {
```

Comment:
If all elements are unused can this apply to the struct as a whole?

Reply:
I suppose it can, I was just thinking as soon as you start referring to each individual struct-field you could slowly remove each warning-suppression?